src/m365/spfx/commands/project/DeployWorkflow.ts

```diff
@@ -13,6 +13,9 @@ export const workflow: GitHubWorkflow = {
   jobs: {
     "build-and-deploy": {
       "runs-on": "ubuntu-latest",
+      env: {
+        NodeVersion: "22.x"
```

Comment:
since I updated the node default value to 22, this PR should get merged after SPFx 1.21 will be GA